src/System.Buffers/src/System/Buffers/Utilities.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace System.Buffers
+{
+    internal static class Utilities
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
My C++ background is shining through here...I haven't run benchmarks around having this attribute vs. not having it. I put it in due to the simplicity and size of the function and around the cost savings of a function call